### PR TITLE
Add SCT and dcm2niix installation instructions

### PR DESCRIPTION
**Pull request by valosekj**
This PR adds instructions on how to install SCT and dcm2niix.

As discussed in https://github.com/sct-pipeline/balgrist-sci/issues/1, we opted to describe the installation process in README (instead of having an "installation script").

Link to the README: https://github.com/sct-pipeline/balgrist-sci/blob/jv/add_installation_instructions/README.md

Related: https://github.com/sct-pipeline/balgrist-sci/issues/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,84 @@
...
+[Spinal Cord Toolbox v6.4](https://github.com/spinalcordtoolbox/spinalcordtoolbox/releases/tag/6.4)
+[dcm2niix >= v1.0.20220505](https://github.com/rordenlab/dcm2niix?tab=readme-ov-file#install)
+
+## Installation
+
+### SCT Installation
+
+1. Open a new terminal:
+
+Press <kbd>command</kbd> + <kbd>space</kbd> and type `Terminal` and press <kbd>return/enter</kbd>.
```
Comment (sandrinebedard) [inline README.md:23]: This is only for macos right? (maybe specify)

File: README.md [hunk 2]
```diff
@@ -2,8 +2,84 @@
...
 3. Quality control (QC) + manual compression level labeling
 4. Lesion metric computation
+
+
+## Dependencies
+
+[Spinal Cord Toolbox v6.4](https://github.com/spinalcordtoolbox/spinalcordtoolbox/releases/tag/6.4)
+[dcm2niix >= v1.0.20220505](https://github.com/rordenlab/dcm2niix?tab=readme-ov-file#install)
+
+## Installation
```
Comment (sandrinebedard) [inline README.md:17]: Consider adding numbers before each steps and a TOC?

File: README.md [hunk 3]
```diff
@@ -2,8 +2,84 @@
...
+> [!NOTE]
+> We are not using `git clone` because Apple Developer Tools are needed for `git`.
+
+3. Check that SCT was installed correctly:
+
+Close the terminal and open a new one (press <kbd>command</kbd> + <kbd>space</kbd> and type `Terminal` and press <kbd>return/enter</kbd>.).
+
+```bash
+sct_check_dependencies
+echo $SCT_DIR
```
Comment (sandrinebedard) [inline README.md:56]:
```suggestion
# Display location of SCT installation
echo $SCT_DIR
```

File: README.md [hunk 4]
```diff
@@ -2,8 +2,84 @@
...
+```
+
+The expected output is `[OK]` for all dependencies.
+
+### dcm2niix Installation
+
+1. Open a new terminal (if you closed the previous one):
+
+Press <kbd>command</kbd> + <kbd>space</kbd> and type `Terminal` and press <kbd>return/enter</kbd>.
```
Comment (sandrinebedard) [inline README.md:65]: Maybe specify this is for macos?

**Review by sandrinebedard — APPROVED** [review]
Looks good @valosekj ! Just minor comments

File: README.md (hunk 1 above)
Comment (valosekj) [reply README.md:31]: you are right! specified in https://github.com/sct-pipeline/balgrist-sci/pull/2/commits/9100c9fdcbb1cb163c95ba1f5fa1d941add64e8d

File: README.md (hunk 4 above)
Comment (valosekj) [reply README.md:75]: 👍🏻 done in https://github.com/sct-pipeline/balgrist-sci/pull/2/commits/9100c9fdcbb1cb163c95ba1f5fa1d941add64e8d

File: README.md (hunk 2 above)
Comment (valosekj) [reply README.md]: good point! TOC added in https://github.com/sct-pipeline/balgrist-sci/pull/2/commits/62b47507786b194edfce216f35e1ea5a714eb1cb

File: README.md (hunk 3 above)
Comment (valosekj) [reply README.md:66]: 👍🏻 fixed in https://github.com/sct-pipeline/balgrist-sci/pull/2/commits/6c9261c3bf96144a3a0e35b2a43b27d5ea4dfad5

**Comment by valosekj** [comment]
Merging to continue with other pipeline steps

File: README.md [hunk 5]
```diff
@@ -2,8 +2,99 @@
...
+2. Run the following commands in the terminal (you can copy-paste the whole block):
+
+> [!NOTE]
+> The installation process will take a few minutes.
+
+```bash
+# Go to your home directory
+cd ~
+# Download SCT v6.4
+wget https://github.com/spinalcordtoolbox/spinalcordtoolbox/archive/refs/tags/6.4.zip
```
Comment (valosekj) [inline README.md:43]: Just a note: I use `wget` instead of `git clone` because Apple Developer Tools are needed for `git`.

**Pull request merged**